src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -177,17 +179,36 @@ function  azure_rdma_preparation()
 azure_rdma_preparation
 {{/ azRDMA }}
 {{/ reqAzRDMA }}
-# Write env to system-wide environment
-env | grep -E "^PAI|PATH|PREFIX|JAVA|HADOOP|NVIDIA|CUDA" > /etc/environment
+
+function write_user_command()
+{
+  PAI_USER_COMMAND_DIR="/pai/taskData"
+  PAI_USER_COMMAND_PATH="${PAI_USER_COMMAND_DIR}/command.sh"
```

Comment:
you can use the path `/pai/command.sh`, maybe make it an argument to functions `write_user_command` and `run_user_command`?